edi_835_parser/transaction_set/transaction_set.py

```diff
@@ -97,6 +97,7 @@ def serialize_service(
 
 		datum = {
 			'marker': claim.claim.marker,
+			'payer_claim_control_number': claim.claim.payer_claim_control_number if claim.claim else None,
```

Comment:
i'm not currently using the pandas integration, but if folks have opinions on which of the additional fields belong in the pandas output i'm happy to make changes